src/pages/Delivery/Delivery.styled.jsx

```diff
@@ -24,10 +24,9 @@ const HeaderMap = styled.div`
 `
 const HeaderText = styled.div`
   text-align: center;
-  display: flex;
-  flex-direction: column;
-  margin-left: 25px;
   height: auto;
+  position: absolute;
+  left: 40% ;
```

Comment:
please take care with percentages without min or max values

@angelik0828 please verify if this is working in other viewports/responsive